tests/ha/hawk_gui.pm

```diff
@@ -83,7 +79,10 @@ sub run {
     assert_script_run "mkdir -m 1777 $path";
     assert_script_run "xhost +";
     barrier_wait("HAWK_GUI_CPU_TEST_START_$cluster_name");
-    my $docker_cmd = "docker run --rm --name test --ipc=host -v /tmp/.X11-unix:/tmp/.X11-unix -e DISPLAY=\$DISPLAY -v \$PWD/$path:/$path ";
+    assert_script_run("ls");
```

Comment:
why `ls` is needed and why with `assert_script_run` and not `enter_cmd`?

Reply:
It is not needed - it is obe of the test outputs to be removed in the final, when I remove the WIP (which is now, just waiting for VRs to be fine)